libcentrifugo/application.go

```diff
@@ -498,7 +498,7 @@ func (app *Application) pubPing() error {
 		Goroutines: runtime.NumGoroutine(),
 		NumCPU:     runtime.NumCPU(),
 		Gomaxprocs: runtime.GOMAXPROCS(-1),
-		Metrics:    *app.metrics,
+		Metrics:    *app.metrics.GetSnapshot(),
```

Comment:
Ah I guess I made mistake here - I thought about it as if this was a pointer to Metrics not a Metrics copy.

I think just making it a pointer probably solves the race since there would be no read of the counter values here.

Then when it comes to marshal the JSON, the Metrics marshalling code takes care of only reading the non-atomic stuff here.

By adding GetSnapshot, we are kind of doing the copy twice, both under lock.

Does just changing this to a `*Metrics` in `NodeInfo` make the race detector happy?